pkger/models.go

```diff
@@ -733,6 +734,8 @@ const (
 	fieldChartYPos          = "yPos"
 	fieldChartBinSize       = "binSize"
 	fieldChartDomain        = "domain"
+	fieldChartBinCount      = "binCount"
```

Comment:
nit: can you add these in alphabetical order, looks like last 3 didn't get in order